google/cloud/storage/tests/testbench.py

```diff
@@ -448,7 +447,76 @@ def get_acl(self, entity):
 
     def update_acl(self, entity, role):
         """
-        Update a single AccessControl entry in this Object revision.
+        Update a single BucketAccessControl entry in this bucket.
+
+        :param entity:str the name of the entity.
+        :param role:str the new role for the entity.
+        :return:dict with the contents of the BucketAccessControl.
+        """
+        return self.insert_acl(entity, role)
+
+    def insert_default_object_acl(self, entity, role):
+        """
+        Insert (or update) a new default ObjectAccessControl entry for this bucket.
+
+        :param entity:str the name of the entity to insert.
+        :param role:str the new role
+        :return:dict the dictionary representing the new ObjectAccessControl.
+        """
+        entity = canonical_entity_name(entity)
+        email = ''
+        if entity.startswith('user-'):
+            email = email.replace('user-', '', 1)
+        # Replace or insert the entry.
+        indexed = {
+            entry.get('entity').lower(): entry
+            for entry in self.metadata.get('defaultObjectAcl', [])
+        }
+        indexed[entity] = {
+            'bucket': self.name,
+            'email': email,
+            'entity': entity,
+            'etag': self.metadata.get('etag', 'XYZ='),
+            'id': self.metadata.get('id', '') + '/' + entity,
+            'kind': 'storage#objectAccessControl',
+            'role': role,
+            'selfLink': self.metadata.get('selfLink') + '/acl/' + entity
+        }
+        self.metadata['defaultObjectAcl'] = indexed.values()
+        return indexed[entity]
+
+    def delete_default_object_acl(self, entity):
+        """
+        Delete a single default ObjectAccessControl entry from this bucket.
+
+        :param entity:str the name of the entity.
+        :return:None
+        """
+        entity = canonical_entity_name(entity)
+        indexed = {
+            acl.get('entity').lower(): acl
```

Comment:
Same comment as above.  Actually, this happens in several places in testbench.py.  I did a lazy search for a large number of spaced followed by "for" and found another 2 places:

````
"          for"
````